### DISCUSSION TITLE
Cannot download wxMaxima 21 instrux

### DISCUSSION THREAD

**Discussion by rich1943** [wxMaxima-developers/wxmaxima#1539] [Q&A]
May 2021 - I just installed v21 of wxMaxima. This site "https://wxmaxima-developers.github.io/wxmaxima/help.html" says there is an 'offline manual' for wxMaxima but it will not download completely with Firefox v68 (latest available FF in slackware). In fact, in earlier versions of wxMaxima I have never been able to get an offline manual for wxMaxima, which would be very helpful.
As it stands, the website seems to be incorrect, unless the download is actually at some other link.
Help obtaining a useful offline manual would be greatly appreciated. Thanks.
jrc

**Comment by rich1943** [comment]
Please note - I'm most grateful for the developers' work on wxMaxima. I find it very useful and wish I had known about it years ago - now retired. I don't have the ability to do that kind of work, for sure. And I know that the program is a work in progress. As well, I'm aware of the other documentation online - tutorials, examples, etc which are useful and much appreciated. I just want to encourage anyone who does have the knowledge to consider making a true 'manual' available. Thanks,
jrc

**Reply by daute** [reply]
Hi,
If you have installed wxMaxima, the manual in HTML format should already be on your harddisk (if you installed into `/usr/local`, usually in the directory: `/usr/local/share/doc/wxmaxima`).

I have no idea what's the problem with the browsers, but I added now PDF versions of the manuals to the website. They should be easy to download (currently no PDFs for the Ukrainian and Chinese translation, I just found a bug with the generation of PDF manuals with foreign (cyrillic and chinese) characters.

Best regards, Wolfgang

**Reply by rich1943** [reply]
Thanks Wolfgang, it is there - actually in /usr/share/doc/wxMaxima-21.01.0 in my Slackware64 14.2. Also downloaded the new pdf you put up, which works as well. So my question/comment is answered, but I don't see how to 'mark' it so. Anyway, many thanks. jrc

**Reply by daute** [reply]
With commit 98da7c6 I fixed the issue with the PDFs of the Ukrainian and Chinese manual translations (which are fairly complete, thanks to our translators), the PDFs are now also downloadable on the website.

**Comment by gunterkoenigsmann** [comment]
On Debian maxima by default comes without any manual, though (at least this happened on my computer) and the file `maxima_singlepage.html` is in no installable debian package - which means that it doesn't come with Ubuntu, neither.

If you wanted to file a bug report against that I wouldn't object. But that is a maxima, not a wxMaxima packaging bug.
